pinot-tools/src/main/resources/conf/pinot-server-log4j.properties

```diff
@@ -14,9 +14,23 @@
 # limitations under the License.
 #
 
-log4j.rootLogger=INFO, serverLog
+log4j.rootLogger=INFO, serverLog, consoleWarn
 
 log4j.appender.serverLog=org.apache.log4j.FileAppender
 log4j.appender.serverLog.layout=org.apache.log4j.PatternLayout
 log4j.appender.serverLog.File=pinotServer.log
-log4j.appender.serverLog.layout.ConversionPattern=%d{yyyy/MM/dd HH:mm:ss.SSS} %p [%c] [%x] %m%n
\ No newline at end of file
+log4j.appender.serverLog.layout.ConversionPattern=%d{yyyy/MM/dd HH:mm:ss.SSS} %p [%c] [%x] %m%n
+
+# Output server starter logs to the console
+log4j.logger.com.linkedin.pinot.server.starter.helix.HelixServerStarter=INFO, consoleLog
```

Comment:
It's needed. This makes it so that we display all warnings on the console but only the info for the server starter, otherwise the console would be really noisy. The logs on disk still contain all the info level logs.

Right now as it is, we don't display any status when starting the controller/broker/server on open source release, so a user has no idea of what happened (or even if anything happened) after start up, with this they get a friendly message that the controller has started and is listening on port 9000.